rpc/types.go

```diff
@@ -33,7 +33,7 @@ type API struct {
 	Namespace     string      // namespace under which the rpc methods of Service are exposed
 	Version       string      // api version for DApp's
 	Service       interface{} // receiver instance which holds the methods
-	Public        bool        // indication if the methods must be considered safe for public use
+	Public        bool        // deprecated - this field is no longer used, but retained for compatibility
```

Comment:
Per a request from @fjl I have retained the field, but marked it as "deprecated".